fullstop-web/fullstop-whitelisting-rules/src/main/java/org/zalando/stups/fullstop/config/RuleControllerProperties.java

```diff
@@ -0,0 +1,25 @@
+package org.zalando.stups.fullstop.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.List;
+
+import static com.google.common.collect.Lists.newArrayList;
+
+@Configuration
+@ConfigurationProperties(prefix = "fullstop.whitelist")
+public class RuleControllerProperties {
+
+    private List<String> allowedTeams = newArrayList();
+
+    public List<String> getAllowedTeams() {
+
+        return allowedTeams; //TODO default values?
```

Comment:
//TODO default values?
I will go for no default, if empty, then no one is able to access it.